swagger.yaml

```diff
@@ -1634,14 +1666,15 @@ definitions:
         type: string
         example: "sample-name"
       service:
-        type: string
-        example: "bigquery"
+        type: array
+        example: "bigquery,postgres"
       description:
         type: string
         example: "sample description"
       data:
         type: object
         description: "dynamic object"
+        example: "config.data[entity]=gojek"
```

Comment:
@scortier I don't think this query like structure belongs to this place. The `config.data[entity]=gojek` should only come in the query param right?

Reply:
Oh, the base asset is not changed, thanks for pointing out, will revert the changes